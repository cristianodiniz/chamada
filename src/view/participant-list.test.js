import React from 'react';
import ParticipantList from './participant-list';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';

describe('Participant List Tests Component', function () {

  it('Should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ParticipantList />, div);
    ReactDOM.unmountComponentAtNode(div);
  });


  test('Should Match Snapshot', () => {
    const component = renderer.create(
      <ParticipantList />,
    );
    
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });


})