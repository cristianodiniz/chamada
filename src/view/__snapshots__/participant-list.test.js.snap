// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Participant List Tests Component Should Match Snapshot 1`] = `
<ul
  className="MuiList-root ParticipantList-root-29 MuiList-padding"
/>
`;
